### PR TITLE
Add new fields in user client for CTE users.

### PR DESCRIPTION
**Pull request by hogo6002**

### CODE REVIEW

**Comment by Lauragra** [comment]
@hogo6002 , please review the content workflow documentation. Before we merge a PR, we have a docs reviewer perform a content review. I've set the appropriate label. If the schema is live, you can set the **schema live** label and we'll merge it as soon as a content reviewer approves.

File: api-reference/v1.0/resources/callrecords-clientuseragent.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ the [userAgent](callrecords-useragent.md) type.
 |headerValue|String|User-agent header value reported by this endpoint.|
 |platform|microsoft.graph.callRecords.clientPlatform|Identifies the platform used by this endpoint. Possible values are: `unknown`, `windows`, `macOS`, `iOS`, `android`, `web`, `ipPhone`, `roomSystem`, `surfaceHub`, `holoLens`, `unknownFutureValue`.|
 |productFamily|microsoft.graph.callRecords.productFamily|Identifies the family of application software used by this endpoint. Possible values are: `unknown`, `teams`, `skypeForBusiness`, `lync`, `unknownFutureValue`, `azureCommunicationServices`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value(s) in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `azureCommunicationServices`.|
+|communicationServiceId|String|Immutable resource identifier of the Azure Communication Service associated with User agents based on Communication Services APIs.|
```
Comment (GageAmes) [inline api-reference/v1.0/resources/callrecords-clientuseragent.md:25]: For consistency with the other descriptions here, consider changing this from "associated with User agents" to "associated with this endpoint"
```suggestion
|communicationServiceId|String|Immutable resource identifier of the Azure Communication Service associated with this endpoint based on Communication Services APIs.|
```

File: api-reference/beta/resources/callrecords-clientuseragent.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ the [userAgent](callrecords-useragent.md) type.
 |headerValue|String|User-agent header value reported by this endpoint.|
 |platform|microsoft.graph.callRecords.clientPlatform|Identifies the platform used by this endpoint. Possible values are: `unknown`, `windows`, `macOS`, `iOS`, `android`, `web`, `ipPhone`, `roomSystem`, `surfaceHub`, `holoLens`, `unknownFutureValue`.|
 |productFamily|microsoft.graph.callRecords.productFamily|Identifies the family of application software used by this endpoint. Possible values are: `unknown`, `teams`, `skypeForBusiness`, `lync`, `unknownFutureValue`, `azureCommunicationServices`.  Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value(s) in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `azureCommunicationServices`.|
+|communicationServiceId|String|Immutable resource identifier of the Azure Communication Service associated with User agents based on Communication Services APIs.|
```
Comment (GageAmes) [inline api-reference/beta/resources/callrecords-clientuseragent.md:27]: For consistency with the other descriptions here, consider changing this from "associated with User agents" to "associated with this endpoint"

```suggestion
|communicationServiceId|String|Immutable resource identifier of the Azure Communication Service associated with this endpoint based on Communication Services APIs.|
```

File: api-reference/beta/resources/callrecords-clientuseragent.md [hunk 3]
```diff
@@ -24,6 +24,8 @@ the [userAgent](callrecords-useragent.md) type.
...
+|azureADAppId|String|The unique identifier for the application that is assigned to an application by Azure AD.|
```
Comment (GageAmes) [inline api-reference/beta/resources/callrecords-clientuseragent.md:28]:
```suggestion
|azureADAppId|String|The unique identifier of the Azure AD application used by this endpoint.|
```

File: api-reference/v1.0/resources/callrecords-clientuseragent.md [hunk 4]
```diff
@@ -22,6 +22,8 @@ the [userAgent](callrecords-useragent.md) type.
...
+|azureADAppId|String|The unique identifier for the application that is assigned to an application by Azure AD.|
```
Comment (GageAmes) [inline api-reference/v1.0/resources/callrecords-clientuseragent.md:26]:
```suggestion
|azureADAppId|String|The unique identifier of the Azure AD application used by this endpoint.|
```

**Review by GageAmes — APPROVED** [review]
I suggested some minor rephrasing of the property descriptions but otherwise this looks good to me. The Graph Content Reviewers may have other suggestions for making the descriptions more clear, so please prioritize their feedback over mine.

**Comment by fafa-npu** [comment]
> 

Hi @Lauragra, we have checked in the PR in ADO for graph schema changes: https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/6282358 . Can we set it to 'schema live' now?

**Comment by Lauragra** [comment]
> > 
> 
> Hi @Lauragra, we have checked in the PR in ADO for graph schema changes: https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/6282358 . Can we set it to 'schema live' now?

Yes you can set schema live. This PR still needs to have a content review and be approved, but it is currently in the queue. Thanks.

**Comment by isvargasmsft** [comment]
Hello @hogo6002, 

Please address the feedback from Gage so I can complete the content review. 

Warm regards, 

Isaac

**Comment by hogo6002** [comment]
> 

Hi @isvargasmsft,

I have just addressed all feedback. The PR is now ready for content review.

Thanks,
Holly

File: api-reference/beta/resources/callrecords-clientuseragent.md [hunk 5]
```diff
@@ -24,6 +24,8 @@ the [userAgent](callrecords-useragent.md) type.
 |headerValue|String|User-agent header value reported by this endpoint.|
 |platform|microsoft.graph.callRecords.clientPlatform|Identifies the platform used by this endpoint. Possible values are: `unknown`, `windows`, `macOS`, `iOS`, `android`, `web`, `ipPhone`, `roomSystem`, `surfaceHub`, `holoLens`, `unknownFutureValue`.|
 |productFamily|microsoft.graph.callRecords.productFamily|Identifies the family of application software used by this endpoint. Possible values are: `unknown`, `teams`, `skypeForBusiness`, `lync`, `unknownFutureValue`, `azureCommunicationServices`.  Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following value(s) in this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `azureCommunicationServices`.|
+|communicationServiceId|String|Immutable resource identifier of the Azure Communication Service associated with this endpoint based on Communication Services APIs.|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/callrecords-clientuseragent.md:27]: Is Communication Services APIs a branded name? If so, would you be so kind to add a reference link? If it is not, and we are referring to any communication services API, then please use small caps. 

Please apply this comment to the v1 topics as well.

Comment (tomaschladek) [reply api-reference/beta/resources/callrecords-clientuseragent.md:27]: Hi @isvargasmsft, the property refers to Azure Communication Services name - https://azure.microsoft.com/en-us/services/communication-services/. From that perspective it is branded name and we can add the link to the description

Comment (hogo6002) [reply api-reference/beta/resources/callrecords-clientuseragent.md]: Added the link to the beta and v1 description.

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**